### DISCUSSION TITLE
Animating a Process

### DISCUSSION THREAD

**Discussion by ea42gh** [ahuang11/ahlive#20] [Q&A] [answered]
I am really intrigued:
One use case I have is to animate the steps of an algorithm,
e.g., Gaussian Elimination.

I could easily produce a set of figures for display.
The easiest would be SVG since I am already set up to do that.

Can I use this tool to display these images in a sequence
without writing more code?

**Comment by ahuang11** [comment]
At the moment, no, but it's not a bad idea to have a utility that globs all the images and produce a GIF!

For now, you could aggregate them using panel; see:
https://ahlive.readthedocs.io/en/latest/customizations/animating.html#animate
and 
https://panel.holoviz.org/reference/widgets/Player.html#widgets-gallery-player

**Comment by ahuang11** [comment, marked as answer]
It's now possible to do with [`enjoyn`](https://enjoyn.readthedocs.io/en/latest/get_started/overview.html)!
